archlinuxcn/wiliwili/PKGBUILD

```diff
@@ -10,32 +10,35 @@ pkgdesc='专为手柄控制设计的第三方跨平台B站客户端'
 arch=('x86_64' 'aarch64')
 url='https://github.com/xfangfang/wiliwili'
 license=('GPL-3.0-or-later')
-depends=('mpv' 'opencc' 'pystring')
-makedepends=('cmake' 'git' 'libxi' 'ninja' 'python' 'wayland-protocols')
+depends=('mpv' 'opencc' 'pystring' 'tinyxml2')
+makedepends=('cmake' 'git' 'libxi' 'libxinerama' 'ninja' 'python' 'wayland-protocols')
 source=("${pkgname}"::"git+${url}.git#tag=v${pkgver}")
-sha512sums=('SKIP')
+b2sums=('SKIP')
 
 prepare() {
   git -C "${srcdir}/${pkgname}" submodule update --init --recursive
-}
 
-build() {
   cmake \
     -S "${srcdir}/${pkgname}" \
     -B "${srcdir}/build" \
-    -G Ninja \
     -D CMAKE_BUILD_TYPE=Release \
     -D CMAKE_INSTALL_PREFIX='/usr' \
     -D INSTALL=ON \
     -D PLATFORM_DESKTOP=ON \
     -D USE_SYSTEM_CURL=ON \
-    -D USE_SYSTEM_OPENCC=ON \
     -D USE_SYSTEM_PYSTRING=ON \
-    -D USE_SYSTEM_SDL2=ON
+    -D USE_SYSTEM_OPENCC=ON \
+    -D USE_SYSTEM_TINYXML2=ON \
+    -D USE_SYSTEM_SDL2=ON \
+    -D GLFW_BUILD_WAYLAND=ON \
+    -D GLFW_BUILD_X11=ON \
+    -G Ninja
+}
 
-  ninja -C "${srcdir}/build" wiliwili
+build() {
+  cmake --build "${srcdir}/build"
 }
```

Comment:
上游那边的源码编译时所产生的 Warning 不因由打包者来解决。

Reply:
I am placing a marker for future code reviews, this will not affect the current code merge. I am not mandating that you address the warnings generated during compilation. However, resolving these compilation warnings or reporting them upstream would be greatly appreciated.